src/routes/rockets/Rockets.js

```diff
@@ -1,9 +1,32 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { fetchAllRockets } from '../../store/rockets/Rockets';
+import RocketCard from '../../components/rocket/rocketCard';
 
-const Rockets = () => (
-  <h1>
-    Ready for launch
-  </h1>
-);
+const Rockets = () => {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchAllRockets());
+  }, []);
```

Comment:
- [ ] **[REQUIRED]** Please make sure to fetch the data from the API only once, at window load!

One way you can do that is by dispatching the **fetchAllRockets** action conditionally if there are no **Rockets** in the **store**:

`if(rockets.length === 0) dispatch(fetchAllRockets());`